### PR TITLE
{bio}[foss/2020b] Myokit v1.32.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#12231~~ (pocl), ~~#12259~~ (SUNDIALS), ~~#12260~~ (lmxl)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3573.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/b008ad944881864519ef04c96f93bbc6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3153.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c8291c71200b442bbf38fc7f2c5827a7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2637.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/476f1034e212ab22652175a658ae8ed9 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.7
See https://gist.github.com/07fbfa768141dff27537e9496ebcdb39 for a full test report.

File: easybuild/easyconfigs/m/Myokit/Myokit-1.32.0-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    ('lxml', '4.6.2'),
+    ('PyQt5', '5.15.1'),
+    ('SUNDIALS', '5.7.0'),
+    ('pocl', '1.6'),
+]
+
+# configparser is part of stdlib in Python 3
+preinstallopts = "sed -i 's/.*configparser.*//g' setup.py && "
+
+sanity_pip_check = True
```
Comment (verdurin) [inline easybuild/easyconfigs/m/Myokit/Myokit-1.32.0-foss-2020b.eb:32]: @boegel missing `download_dep_fail`?

**Review by verdurin — DISMISSED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 11 out of 11 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/8d6723eb473d40645ecef457b50f81e6 for a full test report.

File: easybuild/easyconfigs/m/Myokit/Myokit-1.32.0-foss-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/m/Myokit/Myokit-1.32.0-foss-2020b.eb:33]: fixed in d1e2f30

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 10 out of 13 (1 easyconfigs in total)
node359.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/a6053bb7ceeda2cfccc1c220cdf149cc for a full test report.

**Comment by boegel** [comment]
@lexming `Qt5` failed with `Unable to create file. File name too long`... 🤦

**Comment by lexming** [comment]
@boegel indeed... it's always the same story with `Qt5`... I'll build it first before resubmitting a test report

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/a28812e2cb3cde6b65145ce626775c42 for a full test report.

**Comment by branfosj** [comment]
As @lexming saw a failure in building Qt5 and not this, I am happy to proceed with merging this.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**